common/model-views.cpp

```diff
@@ -32,7 +32,7 @@
 #include "metadata-helper.h"
 #include "calibration-model.h"
 #include "sw-update/http-downloader.h"
-#include "utilities/filesystem/glob.h"
+#include "filesystem/glob.h"
```

Comment:
Not sure this is good... let's discuss

Reply:
You caught the 1 thing I wasn't sure of :)
In order for us to allow 
```
#include  <third-party/filesystem/glob.h>
```

We need the root to be included in the path which is also not good, so since it's only in a tool a preferred making it simple.

What do you think?

Reply:
Will push this to a follow-up PR

Reply:
Wait with the merge, I still got issues using the dispatcher from the tool checking it.